tutorials/kafka-exactly-once/README.md

```diff
@@ -33,6 +33,8 @@ to target/validation.txt
 To run the application on a cluster a running Kafka service is needed.
 A local Kafka single-node instance can easily be deployed (see [kafka.apache.org/quickstart](https://kafka.apache.org/quickstart)).
 
+Kafka topics should be cleared after every run to get a right validation

```

Comment:
Some sample commands on how to clear the topics and naming the specific topics would be helpful.

"to get a right validation output line" => in order for validation to work correctly